interfaces/builtin/common.go

```diff
@@ -55,6 +57,9 @@ type commonInterface struct {
 	connectedPlugUDev      []string
 	rejectAutoConnectPairs bool
 
+	connectedPlugUpdateNSAppArmor string
```

Comment:
Once this lands, we should consider updating the `desktop` iface to use `commonInterface`

Reply:
I'll prepare a follow-up as this is more and more likely to land.

Reply:
#8134 is the first part of the follow-up